web/app/js/components/MetricsTable.jsx

```diff
@@ -44,44 +44,42 @@ const httpStatColumns = [
     dataIndex: "successRate",
     isNumeric: true,
     render: d => <SuccessRateMiniChart sr={d.successRate} />,
-    sorter: (a, b) => numericSort(a.successRate, b.successRate)
+    sorter: d => d.successRate
   },
   {
     title: "RPS",
     dataIndex: "requestRate",
     isNumeric: true,
     render: d => metricToFormatter["NO_UNIT"](d.requestRate),
-    sorter: (a, b) => numericSort(a.requestRate, b.requestRate)
+    sorter: d => d.requestRate
   },
   {
     title: "P50 Latency",
     dataIndex: "P50",
     isNumeric: true,
     render: d => metricToFormatter["LATENCY"](d.P50),
-    sorter: (a, b) => numericSort(a.P50, b.P50)
+    sorter: d => d.P50
   },
   {
     title: "P95 Latency",
     dataIndex: "P95",
     isNumeric: true,
     render: d => metricToFormatter["LATENCY"](d.P95),
-    sorter: (a, b) => numericSort(a.P95, b.P95)
+    sorter: d => d.P95
   },
   {
     title: "P99 Latency",
     dataIndex: "P99",
     isNumeric: true,
     render: d => metricToFormatter["LATENCY"](d.P99),
-    sorter: (a, b) => numericSort(a.P99, b.P99)
+    sorter: d => d.P99
   },
   {
     title: "TLS",
     dataIndex: "tlsRequestPercent",
     isNumeric: true,
     render: d => _isNil(d.tlsRequestPercent) || d.tlsRequestPercent.get() === -1 ? "---" : d.tlsRequestPercent.prettyRate(),
-    sorter: (a, b) => numericSort(
-      a.tlsRequestPercent ? a.tlsRequestPercent.get() : -1,
-      b.tlsRequestPercent ? b.tlsRequestPercent.get() : -1)
+    sorter: d => d.tlsRequestPercent ? d.tlsRequestPercent.get() : -1
```

Comment:
TIOLI: This statement is equivalent as well d.tlsRequestPercent || -1

Reply:
I don't think this is true, since `tlsRequestPercent` is a `Percentage` so to get the value out of it, you have to call `get()`